### DISCUSSION TITLE
Some guidance regarding <model-viewer> and filament ?

### DISCUSSION THREAD

**Discussion by Arditiww1** [google/filament#3922] [General]
I am new to this 3D web rendering world, can somebody help when should you use filament or <model-viewer>. Please Help as both can be used in mobile and web.

**Comment by prideout** [comment]
Personally, I recommend Filament for mobile apps, native desktop apps, or when you want to use the same code for both native and web.

However if you need a viewer only for the web, I'd actually recommend `<model-viewer>`.  It is very easy to use and works across a broad swath of browsers.  That's my opinion anyway.  :)

**Reply by Arditiww1** [reply]
Thanks for your reply and help.
